packages/python/build.sh

```diff
@@ -98,7 +99,7 @@ termux_step_create_debscripts() {
 
 	if [[ -f "$TERMUX_PREFIX/bin/pip" && \
 	 ! (("$TERMUX_PACKAGE_FORMAT" = "debian" && -f $TERMUX_PREFIX/var/lib/dpkg/info/python-pip.list) || \
-	    ("$TERMUX_PACKAGE_FORMAT" = "pacman" && -d $TERMUX_PREFIX/var/lib/pacman/local/python-pip-*)) ]]; then
+	    ("$TERMUX_PACKAGE_FORMAT" = "pacman" && \$(ls $TERMUX_PREFIX/var/lib/pacman/local/python-pip-*))) ]]; then
```

Comment:
Does not this lead to a distracting message such as

```
ls: cannot access '...': No such file or directory
```

when `$TERMUX_PREFIX/var/lib/pacman/local/python-pip-*` does not match.

I would suppose

```bash
ls ... 2>/dev/null
```

or using `echo` with precedent `shopt -s nullglob`.

Reply:
I checked, this check does not return the given message.

Reply:
OK then.